Mathlib/Combinatorics/Additive/QuotientInterGrowth.lean

```diff
@@ -0,0 +1,80 @@
+/-
+Copyright (c) 2024 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import Mathlib.Algebra.Group.Pointwise.Finset.Basic
+import Mathlib.Combinatorics.Enumerative.DoubleCounting
+import Mathlib.GroupTheory.QuotientGroup.Defs
+
+/-!
+# Growth in the quotient and intersection with a subgroup
+
+For a group `G` and a subgroup `H ≤ G`, this file upper and lower bounds the growth of a finset by
+its growth in `H` and `G ⧸ H`.
+-/
+
+open Finset Function
+open scoped Pointwise
+
+namespace Finset
+variable {G : Type*} [Group G] [DecidableEq G] {H : Subgroup G} [DecidablePred (· ∈ H)] [H.Normal]
+  {A : Finset G} {m n : ℕ}
+
+@[to_additive]
+lemma card_pow_quotient_mul_pow_inter_subgroup_le :
```

Comment:
I agree this is morally an `inter`, but if we're using this naming convention then I'm not sure `inter` is appropriate here...

Reply:
I'm following the naming conventions from the `GroupTheory.QuotientGroup` folder, but I agree they aren't great... I don't have a better idea here though